src/ui/components/NumberField.qml

```diff
@@ -21,6 +21,8 @@ TextField {
     property string keyframe: "";
     property bool keyframesEnabled: false;
     property real finalValue: value;
+    
+    property var contextMenu: defaultContextMenu;

```

Comment:
Can you test if `property Menu contextMenu` works too? it would be preferable instead of var (var goes through QVariant)

Reply:
works